agent/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:16.04
```

Comment:
why?

Reply:
At one point I thought that the agent might get confused by the image it was running on, and the image that corresponds to the mongod that was expected to be running, so I changed everything to ubuntu16. Now I'm not sure if this is a situation we should care about, and I'll remove it for now.

Reply:
actually I think it is better if we use ubuntu16 everywhere, just to make sure this is not a variable we should care about, for now. And then we can be specific on the base images.